containers/python-examples/NLTK/notebooks/notebooks/01-example.py

```diff
@@ -0,0 +1 @@
+print("Hello world!")
```

Comment:
Do we want a more DH specific example?

Reply:
Yes. The example will likely vary depending on which deployment is picked.

Everything should come with at least one base DH example. Containers w/ examples should come w/ scripts to work on local data, and AI containers should come w/ a script that uses that particular package. Probably the iris stuff.